### ISSUE TITLE
More contrast please

### ISSUE THREAD

**Issue by laktak** [sebst/pythonic-news#19]
The text is somewhat hard to read on the background. Could you increase the contrast?

See https://news.ycombinator.com/item?id=21818220

**Comment by softwarelimits** [comment]
yes, please avoid that low contrast epidemic. It is not a signal for good quality.
